src/student/student.service.ts

```diff
@@ -135,4 +136,34 @@ export class StudentService {
 
         return cv;
     }
+
+    async studentsSelectedByHr(hrId: string) {
+        try {
+            const hr = await HrEntity.findOne({where: { id: hrId}});
+            if(!hr){
+                return {
+                    actionStatus: false,
+                    message: 'Nie istnieje w bazie HR o takim id',
+                    students: [],
+                }
+            }
+
+            const students = await StudentEntity.findByHrId(hrId);
```

Comment:
Nie wiedziałem że jest taka metoda.
Tworzy się po dodaniu relacji między tabelami?

Reply:
To akurat podpatrzylem z dokumentacji TypeOrm (https://orkhan.gitbook.io/typeorm/docs/active-record-data-mapper), że można sobie stworzyć statyczną funkcję w entity i tak zrobiłem. Stworzyłem funkcję findByHrId w student.entity.ts, a w tym miejscu z niej skorzystałem :)

Reply:
Dzięki, potem zacząłem się zastanawiać czy czasem nie dodałeś metody/funkcji do klasy entity, ale już nie sprawdzałem, sam pomysł fajny zamiast jak ja pobierać wszystko z relacjami i odczytywać z relacji